Setup/deployshield-dev.sh

```diff
@@ -8,7 +8,7 @@
 NETWORK_INTERFACE='eth0'
 SINGLE_MODE=true
 STACK_NAME='shield'
-ES_YML_FILE=docker-compose.yml
+ES_YML_FILE=docker-compose_dev.yml
```

Comment:
Need to add a arg parameter -jenkins to use:
ES_YML_FILE=docker-compose_dev.yml 
otherwise
+ES_YML_FILE=docker-compose.yml